### PR TITLE
Update conf.py to display the announcement banner

### PR DESCRIPTION
**Pull request by jni**
# References and relevant issues

Together with #341, this PR supersedes #332.

# Description

The announcement banner text html was added in #341. This PR now points our
config to it, allowing the banner to be displayed.

Note: #332 also has a CSS fix. However, that fix is included in the latest
release of the napari-sphinx-theme, so I think it's best to leave it out of
this PR. (I would be happy to make a separate PR for it if we don't want to
wait until the dependencies are updated.)

### CODE REVIEW

**Comment by jni** [comment]
Screenshot 2024-01-31 at 10 43 03 pm

Boom. 😃

**Comment by jni** [comment]
Hmm, anyone else seeing this? On Firefox, the link target for the survey link is just a few pixels tall:

https://github.com/napari/docs/assets/492549/44781ef4-dc3a-49e1-85fa-b790a093518a

Fine on Safari and Chrome.

I don't think it has anything to do with this PR and everything to do with the theme, so I don't think we should hold this up to investigate. (I still managed to click it 😂 and I think Firefox users are a persistent bunch! 😂)

**Comment by Czaki** [comment]
On my machine, it looks bad:
obraz

Both Chrome and Firefox

**Comment by jni** [comment]
@Czaki that is this note:

> Note: https://github.com/napari/docs/pull/332 also has a CSS fix. However, that fix is included in the latest
> release of the napari-sphinx-theme, so I think it's best to leave it out of
> this PR. (I would be happy to make a separate PR for it if we don't want to
> wait until the dependencies are updated.)

I will make the separate PR, but the permanent fix is napari/napari-sphinx-theme#145, which is in v0.3.2 (released, but not updated in pins).

**Comment by jni** [comment]
@Czaki see #344. It should be safe to merge before this one, and then main can be merged here.

**Comment by Czaki** [comment]
There are [#6562](https://github.com/napari/napari/pull/6562) and [#6632](https://github.com/napari/napari/pull/6632) that will shortly be ready to merge so #344 will be obsolete?

**Comment by jni** [comment]
Yes, just wanted to make sure you had multiple options at hand before I went to bed. 😊 Feel free to close #344 if it gets fixed! But this one should get merged/backported either way. 😅

**Comment by melissawm** [comment]
Confirming, looks ok on my firefox/linux:


https://github.com/napari/docs/assets/3949932/db324255-aaaa-4db0-8539-867f7cd22d3f

**Review by Czaki — APPROVED** [review]

**Pull request merged**
